sgt-back-end/index.js

```diff
@@ -0,0 +1,164 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error has occurred'
+      });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const name = req.body.name;
+  const course = req.body.course;
+  const score = Number(req.body.score);
+
+  if (!name || !course || !score) {
+    res.status(400).json({
+      error: 'Name, course, and score are required fields!'
+    });
+    return;
+  } else if (score > 100 || score < 0 || !Number.isInteger(score)) {
+    res.status(400).json({
+      error: 'Score must be a number between 0 and 100!'
+    });
+    return;
+  }
+
+  const sql = `
+    insert into "grades" ("name", "course", "score")
+      values ($1, $2, $3)
+      returning *
+  `;
+
+  const params = [name, course, score];
+
+  db.query(sql, params)
+    .then(result => {
+      const grade = result.rows;
```

Comment:
Here `grade` will be an array with a single `grade` in it, but we want to just send the grade object back without the array.
```JAVASCRIPT
const grade = result.rows[0];
```